### PR TITLE
Layout changes to the generated code

### PR DESCRIPTION
**Pull request by jwillemsen**
* ridlbe/c++11/config/cxx_type.rb:
    * ridlbe/c++11/templates/cli/src/enum_anyop.erb:
    * ridlbe/c++11/templates/cli/src/interface_anyop.erb:
    * ridlbe/c++11/templates/cli/src/struct_anyop.erb:
    * ridlbe/c++11/templates/cli/src/typedef_anyop.erb:
    * ridlbe/c++11/templates/cli/src/union_anyop.erb:
    * ridlbe/c++11/templates/cli/src/value_anyop.erb:

### CODE REVIEW

File: ridlbe/c++11/templates/cli/src/union_anyop.erb [hunk 1]
```diff
@@ -11,14 +11,11 @@ void operator<<= (TAOX11_CORBA::Any &any, <%= scoped_cxx_in_type %> _union )
 % else
       TAOX11_NAMESPACE::CDR_Marshal_Policy
 % end
-        >::insert_copy (
-          any,
-          <%= scoped_cxx_typecode %>,
-          _union);
+        >::insert_copy (any, <%= scoped_cxx_typecode %>, _union);
```
Comment (mcorino) [inline ridlbe/c++11/templates/cli/src/union_anyop.erb]: Why do you feel this kind of change is an improvement?

Comment (jwillemsen) [reply ridlbe/c++11/templates/cli/src/union_anyop.erb]: Was looking through the generated code and with anytypecode enabled this adds a lot of sloc without a real value, so just passed all arguments in one line

Comment (mcorino) [reply ridlbe/c++11/templates/cli/src/union_anyop.erb]: Yes, but <%= scoped_cxx_typecode %> can expand to quite a loooong name making a single line a lot less readable. Why this obsession with sloc?

Comment (jwillemsen) [reply ridlbe/c++11/templates/cli/src/union_anyop.erb]: Much wider screen then high ;-)

**Pull request merged**